src/emails/components/EmailHero.tsx

```diff
@@ -21,14 +21,7 @@ export const EmailHero = (props: Props) => {
 
   return (
     <mj-wrapper padding="24px 16px">
-      <mj-section
-        padding="10px 12px"
-        background-color="#e4d2ff"
-        background-url={`${process.env.SERVER_URL}/images/email/hero-bg-gradient.png`}
-        background-repeat="repeat"
-        background-position-x={0}
-        border-radius="16px 16px 0 0"
-      >
+      <mj-section css-class="hero_background">
```

Comment:
I think you missed the padding and radius here:

![image](https://github.com/user-attachments/assets/d64e629f-9e49-4aaa-b258-3305106e122b)

```suggestion
      <mj-section css-class="hero_background"
        padding="10px 12px"
        border-radius="16px 16px 0 0"
      >
```